ghcup-prereleases-0.0.8.yaml

```diff
@@ -2295,12 +2295,166 @@ ghcupDownloads:
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241114/ghc-9.12.0.20241114-x86_64-deb9-linux.tar.xz
+            '>= 12':
+              dlHash: d8ebf6666e00fdafcb1acc5eac39e99fbea4e06b15c23bd33e4f94481b147005
+              dlOutput: ghc-9.12.0.20241114-x86_64-linux-deb12.tar.xz
+              dlSubdir: ghc-9.12.0.20241114-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241114/ghc-9.12.0.20241114-x86_64-deb12-linux.tar.xz
+            unknown_versioning: *ghc9121alpha3-x86_64-deb11
+          Linux_Fedora:
+            '>= 33': &ghc9121alpha3-x86_64-fedora33
+              dlHash: 9e25d7dc3f332aa6ba45aec535ccde843b46b272407e3828f1216959e6e0e586
+              dlOutput: ghc-9.12.0.20241114-x86_64-linux-fedora33.tar.xz
+              dlSubdir: ghc-9.12.0.20241114-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241114/ghc-9.12.0.20241114-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc9121alpha3-x86_64-centos7
+          Linux_Mint:
+            < 20: &ghc9121alpha3-x86_64-ubuntu18_04
+              dlHash: 2333df8b2f4363ec8a78c92d0f0939fdbea64637eb528f32d7581bfc1a0748a3
+              dlOutput: ghc-9.12.0.20241114-x86_64-linux-ubuntu18_04.tar.xz
+              dlSubdir: ghc-9.12.0.20241114-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241114/ghc-9.12.0.20241114-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc9121alpha3-x86_64-ubuntu22_04
+              dlHash: a0dc0c19509a738acfe3b75d284bda1b2e4a07e43b36a46f9f305985372028eb
+              dlOutput: ghc-9.12.0.20241114-x86_64-linux-ubuntu20_04.tar.xz
+              dlSubdir: ghc-9.12.0.20241114-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241114/ghc-9.12.0.20241114-x86_64-ubuntu20_04-linux.tar.xz
+            unknown_versioning: *ghc9121alpha3-x86_64-ubuntu22_04
+          Linux_RedHat:
+            < 9: *ghc9121alpha3-x86_64-centos7
+            '>= 9': *ghc9121alpha3-x86_64-fedora33
+            unknown_versioning: *ghc9121alpha3-x86_64-fedora33
+          Linux_Ubuntu:
+            ( >= 16 && < 18 ): *ghc9121alpha3-x86_64-deb9
+            ( >= 18 && < 19 ): *ghc9121alpha3-x86_64-ubuntu18_04
+            ( >= 19 && < 21 ): *ghc9121alpha3-x86_64-ubuntu22_04
+            ( >= 21 ):
+              dlHash: e0c23a5ddc8b200b019e365248b305767e96d174f0c4fdef568049043d9cbbca
+              dlOutput: ghc-9.12.0.20241114-x86_64-linux-ubuntu22_04.tar.xz
+              dlSubdir: ghc-9.12.0.20241114-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241114/ghc-9.12.0.20241114-x86_64-ubuntu22_04-linux.tar.xz
+            unknown_versioning: *ghc9121alpha3-x86_64-ubuntu22_04
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 256e7390f93e2141d4ada05ecedc57edbf557f809bd54c1cc2589ed2f2a079b0
```

Comment:
The [SHA256SUM](https://downloads.haskell.org/~ghc/9.12.1-alpha3/SHA256SUMS) says the hash is `39652aa4f9d51debaabe02baf46cb136e0a890fafcc25be5ae6ccd0de953048b`. Trying to download this bindist seems to confirm that the 256e hash is wrong.

Reply:
It looks like all hashes are wrong and were copy pasted from the previous prerelease.